pkg/controller/util/solr_util.go

```diff
@@ -188,7 +196,8 @@ func GenerateStatefulSet(solrCloud *solr.SolrCloud, ingressBaseDomain string, ho
 									Name: "POD_HOSTNAME",
 									ValueFrom: &corev1.EnvVarSource{
 										FieldRef: &corev1.ObjectFieldSelector{
-											FieldPath: "metadata.name",
+											FieldPath:  "metadata.name",
+											APIVersion: "v1",
```

Comment:
Does this need the trailing comma?

Reply:
I am not 100% sure about this but I think trailing comma is allowed here because of http://www.relaxedjson.org/ spec.

Reply:
to answer "Is it needed?" there are some benefits to having a trailing comma - https://recology.info/2019/02/trailing-commas/

Reply:
Yeah I think my VS Code linter is adding them, so I guess we stick with them